src/apkAnalyzer/CiRunner.ts

```diff
@@ -42,12 +42,26 @@ export interface CiCore {
 export default class CiRunner {
     ciCore: CiCore; // either Github action core or ADO Task
     thresholdChecker: ThresholdChecker;
-    telemetryClient: appInsights.TelemetryClient;
 
     constructor(ciCore: CiCore) {
         this.ciCore = ciCore;
         this.thresholdChecker = new ThresholdChecker(ciCore);
+    }
+
+    /**
+     * Runs the ci runner with telemetry enabled if set     
+     */
+    public async run() {
+        const telemtryEnabled = this.ciCore.getInput('telemetryEnabled');
 
+        if (telemtryEnabled != 'false') {
```

Comment:
Can we do "if (!telemetryEnabled)"? (or "if (telemetryEnabled)" and reverse the statements)

Reply:
Done